pkgs/tools/system/osquery/platform-nixos.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/tools/get_platform.py b/tools/get_platform.py
+index 3dd34516..f53ca83a 100644
+--- a/tools/get_platform.py
++++ b/tools/get_platform.py
+@@ -26,6 +26,8 @@ DEBIAN_VERSION = "/etc/debian_version"
+ GENTOO_RELEASE = "/etc/gentoo-release"
+ 
+ def _platform():
++    return ("nixos", "nixos")
```

Comment:
To bad that they have no support for `/etc/os-release`. The only disadvantage of this patch is, that it won't work on non-nixos. Also I don't how many people would use osqueryd that way.